spec/ParseObject.spec.js

```diff
@@ -268,18 +268,13 @@ describe('Parse.Object testing', () => {
     });
   });
 
-  it_exclude_dbs(['postgres'])("can set null", function(done) {
+  fit("can set null", function(done) {
```

Comment:
One way to mimic this behavior for PRs would be to create this as a custom script and call it in travis (just like any other test)